ToDoYandex/Extension/Color+Hex.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftUI
+
+extension Color {
+    func toHex() -> String {
+        guard let components = UIColor(self).cgColor.components else {
+            return "#000000"
+        }
+
+        let r = Int(components[0] * 255)
+        let g = Int(components[1] * 255)
+        let b = Int(components[2] * 255)
+        let hex = String(format: "#%02X%02X%02X", r, g, b)
+
+        return hex
+    }
```

Comment:
Если функция ничего не принимает и по факту является строковым значение цвета почему бы не быть `var hexString: String { }`

Reply:
Сейчас этот функционал применяется только в одном месте, в дальнейшем, да и в целом, можно переиспользовать этот конвертер. Так же решил вынести сразу в расширение

Reply:
Я не говорил, что надо убрать из ext или что ты написало неправильно. Лишь то что у любого цвета можно однозначно вычислить hexColor без доп параметров. Это уже дело вкуса, через переменную или функцию. Разницы нет. Но мне просто кажется логичней, так:
```suggestion
    var hexString: String {
        guard let components = UIColor(self).cgColor.components else {
            return "#000000"
        }

        let r = Int(components[0] * 255)
        let g = Int(components[1] * 255)
        let b = Int(components[2] * 255)
        let hex = String(format: "#%02X%02X%02X", r, g, b)

        return hex
    }
```